SWLOR.Game.Server/Native/ResolveAttackRoll.cs

```diff
@@ -301,35 +302,51 @@ private static void OnResolveAttackRoll(void* thisPtr, void* pTarget)
 
```

Comment:
There's a potential NRE on line 255. Mentioned in discord but adding a comment here to note it.